packages/taiko-client/integration_test/test_env.sh

```diff
@@ -23,6 +23,13 @@ export TEST_ACCOUNT_PRIVATE_KEY=0x7c852118294e51e653712a81e05800f419141751be58f6
 export TREASURY=0x1670010000000000000000000000000000010001
 export VERBOSITY=${VERBOSITY-3}
 
+# Preconf p2p variables.
+export PRECONFIRMATION_P2P_DISCOVERY_PATH=memory
```

Comment:
why need to add these extra ENVs?

Reply:
Yeah, there's no error in Ci, if we use memory there will be no more folders(opnode_discovery_db) locally.